addons/purchase/models/purchase_order.py

```diff
@@ -1307,14 +1311,24 @@ def _update_order_line_info(
                 'product_qty': quantity,
                 'sequence': self._get_new_line_sequence(child_field, section_id),
             })
-            if pol.selected_seller_id:
-                # Fix the PO line's price on the seller's one.
-                seller = pol.selected_seller_id
-                price = seller.price_discounted
+            params = {'order_id': self}
```

Comment:
Since this isn't reused, you don't need it stored in a variable, just pass the dict directly as the params value (I know there's another case like this in the same file, but no need to repeat past mistakes)